arviz/tests/base_tests/test_data_zarr.py

```diff
@@ -76,24 +80,24 @@ def test_io_method(self, data, eight_schools_params, store, fill_attrs):
             # InferenceData method
             if store == 0:
                 # Tempdir
-                store = inference_data.to_zarr(store=None)
+                store = to_zarr(inference_data, store=None)
```

Comment:
the test should do something similar to this (a simpler one is probably fine), but be a different one. This test function is called test_io_method and it should continue to do so, the facade functions should have their own test function